### PR TITLE
clipboard overwrite mode

### PR DESCRIPTION
**Pull request by fuku5**
With option --overwrite, the text on the clipboard is overwritten with translated sentences.
I tested this option only on MacBook.

### CODE REVIEW

File: python/gtransweb_gui.py [hunk 1]
```diff
@@ -42,13 +42,22 @@ def get_clipboard_text(clip_mode):
...
+
 class ClipboardChangedHandler():
 
-    def __init__(self, clip_mode, window, buf_time):
+    def __init__(self, clip_mode, window, buf_time, overwrite):
         self.clip_mode = clip_mode
         self.window = window
         self.buf_time = buf_time
         self.query_deq = collections.deque()
+        self.lock = False
```
Comment (takiyu) [inline python/gtransweb_gui.py]: Use more meaningful name.

File: python/gtransweb_gui.py [hunk 2]
```diff
@@ -42,13 +42,22 @@ def get_clipboard_text(clip_mode):
...
+        self.overwrite_clipboard = overwrite
```
Comment (takiyu) [inline python/gtransweb_gui.py]: Do not use a verb for a variable.

File: python/gtransweb_gui.py [hunk 3]
```diff
@@ -59,6 +68,10 @@ def __call__(self, mode):
     def _translate(self):
         # Disable old query
         called_time = self.query_deq.pop()
+        if self.translation_skip:
```
Comment (takiyu) [inline python/gtransweb_gui.py]: Add comment.

File: python/gtransweb_gui.py [hunk 4]
```diff
@@ -75,7 +88,10 @@ def _translate(self):
             window.show_at_cursor()
 
         # Translate clipboard text
-        self.window.translate(src_text)
+        tgt_text = self.window.translate(src_text)
+        if self.overwrite_clipboard:
```
Comment (takiyu) [inline python/gtransweb_gui.py]: Add comment.

**Comment by fuku5** [comment]
changed

**Comment by takiyu** [comment]
Merged.　Thanks.

**Pull request merged**